client/blocks/get-apps/mobile-download-card.jsx

```diff
@@ -162,16 +169,84 @@ class MobileDownloadCard extends Component {
 		return this.props.hasUserSettings && this.props.hasLoadedAccountRecoveryPhone;
 	}
 
-	render() {
-		const { translate } = this.props;
+	getAppStoreBadges() {
+		const { isiPad, isiPod, isiPhone } = userAgent;
+		const isIos = isiPad || isiPod || isiPhone;
+
+		return (
+			<div className="get-apps__badges jetpack">
+				<p className="get-apps__card-text jetpack">
+					{ translate(
+						'Everything you need to publish, manage, and grow your site anywhere, any time.'
+					) }
+				</p>
+				<AppsBadge
+					storeName={ isIos ? 'ios' : 'android' }
+					utm_source={ isIos ? 'calypso-get-apps-button' : 'calypso-get-apps' }
+				/>
+			</div>
+		);
+	}
+
+	getQrCode() {
+		return (
+			<div className="get-apps__qr-code-subpanel jetpack">
+				<QRCode
+					value={ localizeUrl( 'https://apps.wordpress.com/mobile/?campaign=calypso-qrcode-apps' ) }
```

Comment:
Aha, you're right. The current URL is just redirecting to the static apps.wordpress.com site on mobile, rather than the app stores. I also didn't find any results when searching our accounts for the campaign name, so your suspicion that's not supported with this current format seems right. 

I've gone ahead to update the URL (for both the WordPress and Jetpack panels) in https://github.com/Automattic/wp-calypso/pull/69674/commits/ed05e141e9689686876d499ec5041b53a82be1db. I used the `apps.wordpress.com/get?campaign=` structure that we're using in other places, which I believe is what we want.

In addition, this comment made me realise that I should have likely associated the new text link with a campaign. I've done that in https://github.com/Automattic/wp-calypso/pull/69674/commits/ecf94df2373ba79ceb4a8361a1b3a13ddaee2cf0.

Let me know if you think any other changes are needed to this and thanks for spotting this!